### PR TITLE
ROSA First Release - Release Notes Tracker

### PR DESCRIPTION
**Pull request by jonquilwilliams**
Please leave comments here for anything that should be highlighted in the ROSA release notes for this first release.

Please note that it is not possible to generate a preview at this time. Please review the content within the PR and make comments here.

Thank you!

### CODE REVIEW

**Comment by acathrow** [comment]
Under Cluster status notification we also send notifications to the any contacts added to the support tab.

Autoscaling
Do we need the line about hive, I'm not sure users will understand what hive is.

Upgrades.
Can we add something about user being able to select their upgrades strategy (either in OCM or the CLI) which could be automatic or manual.

Security / Auth provider
We do not currently allow the user of htpasswd in the cll (there is RFE to enable it [SDE-985](https://issues.redhat.com/browse/SDE-985)) The create cluster admin command creates a hidden htpasswd IdP but that's obfuscated from the user.

Priviledged containers are supported on ROSA with cluster admin

**Comment by jonquilwilliams** [comment]
@acathrow Thank you for the feedback. Your changes have been made.

File: assemblies/rosa-rn-about.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+:context: rosa-rn-about
+
+[role="_abstract"]
+Release notes overview about {product-title}.
+
+toc::[]
+
+{product-title} (ROSA) is a fully-managed OpenShift Container Platform service, jointly managed and supported by Red Hat and Amazon Web Services (AWS). ROSA provides developers and IT organizations with a hybrid cloud application platform for deploying both new and existing applications on secure, scalable resources with minimal configuration and management overhead.
+ROSA supports a wide selection of programming languages and frameworks, such as Java, JavaScript, Python, Ruby, and PHP.
+Built on Red Hat Enterprise Linux and Kubernetes, ROSA provides a more secure and scalable multi-tenant operating system for today’s enterprise-class applications, while delivering integrated application runtimes and libraries. ROSA enables organizations to meet security, privacy, compliance, and governance requirements.
```
Comment (wgordon17) [inline assemblies/rosa-rn-about.adoc]: > Built on Red Hat Enterprise Linux

It this accurate? I thought everything was running on RHCOS?

File: assemblies/rosa-rn-new-features.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+= New features
+
+:context: rosa-rn-new-features
+
+[role="_abstract"]
+New features for the current release of {product-title}.
+
+toc::[]
+
+This release includes the following new features. For more information, see the Service Definition under Policies.
```
Comment (wgordon17) [inline assemblies/rosa-rn-new-features.adoc:13]: > see the Service Definition under Policies

Should this be a direct link with an anchor tag?

File: modules/rosa-rn-about.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-about.adoc
+
+[id="rosa-rn-about_{context}"]
+= About this release
+
+This is the first release of {product-title}. Customers get the full range of OpenShift Container Platform features, except for the managed services features that are managed and supported by Red Hat and Amazon Web Services (AWS).
```
Comment (wgordon17) [inline modules/rosa-rn-about.adoc]: @acathrow Do want to be publicly saying this?

> This is the first release

Maybe call it a "GA release" or something...first release just sounds too "beta" to me

Comment (wgordon17) [inline modules/rosa-rn-about.adoc]: > except for the managed services features

Do we have this called out anywhere that we can point customers to?

File: modules/rosa-rn-acct-management.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-acct-management_{context}"]
+= Account management
+
+[id="rosa-rn-billing_{context}"]
+== Billing
+
+{product-title} (ROSA) is billed through Amazon Web Services (AWS) based on usage of AWS components used by the service, such as load balancers, storage, EC2 instances, and so on.
```
Comment (wgordon17) [inline modules/rosa-rn-acct-management.adoc]: > based on usage of AWS components

Do we have a component listing that we can point customers to? Similar to https://www.openshift.com/dedicated/ccs#aws-v4-infra

File: modules/rosa-rn-acct-management.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+[id="rosa-rn-self-service_{context}"]
+== Cluster self-service and consoles
+
+Customers can create, scale, and delete their clusters either from OpenShift Cluster Manager (OCM) or the `rosa` CLI utility.
```
Comment (wgordon17) [inline modules/rosa-rn-acct-management.adoc]: > OpenShift Cluster Manager (OCM)

Do we have this defined anywhere in the docs that we can point customers to? Or at least point them directly to https://cloud.redhat.com/openshift? My concern is customers will have no idea what OCM is

File: modules/rosa-rn-acct-management.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+[id="rosa-rn-compute_{context}"]
+== Compute
+
+ROSA supports single availability zone and multiple availability zone clusters. Worker nodes must all be the same instance type and size within a single ROSA cluster.
```
Comment (wgordon17) [inline modules/rosa-rn-acct-management.adoc]: @acathrow This isn't true, right? I'm not familiar with the status of MachinePools on ROSA, I at least _know_ it's not the case for OSD CCS

> Worker nodes must all be the same instance type and size within a single ROSA cluster

File: modules/rosa-rn-acct-management.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+[id="rosa-rn-support_{context}"]
+== Support
+
+ROSA includes Red Hat Premium Support, which can be accessed by using the link:https://access.redhat.com/support[Red Hat Customer Portal] or through link:https://aws.amazon.com/premiumsupport/[AWS Support].
```
Comment (wgordon17) [inline modules/rosa-rn-acct-management.adoc]: While ROSA _does_ include Red Hat Premium Support, I believe the level of AWS support is _optional_, right? So we may want to call that out here. Also, is there anything the customer has to do in order to access the Red Hat support, like a prerequisite? E.g., if they're not a Red Hat customer, do they need to first create a Red Hat account and link it to their AWS account somehow?

File: modules/rosa-rn-logging.adoc [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-logging_{context}"]
+= Logging
+
+{product-title} (ROSA) includes an optional logging stack based on Elasticsearch, Fluentd, and Kibana (EFK). The logging stack in OpenShift Container Platform is designed for short-term retention to aid application troubleshooting, not for long-term log archiving.
+
+Cluster audit logging and application logging are supported. Application logging includes optional log forwarding using the CloudWatch add-on service. Add-on services are configured using the `rosa` command-line.
```
Comment (wgordon17) [inline modules/rosa-rn-logging.adoc]: > Cluster audit logging [...] supported

Is this true? It was my understanding that audit logs we're _only_ available to SREP, and can be requested via a support case, based on https://www.openshift.com/products/dedicated/service-definition#cluster-audit-logging

Comment (wgordon17) [reply modules/rosa-rn-logging.adoc]: > Application logging includes optional log forwarding

I would probably move this into the above section, in order to keep all Application Logging related stuff together.

Comment (wgordon17) [reply modules/rosa-rn-logging.adoc]: Also, this wording seems a little too contradictory

> (ROSA) includes an optional logging stack

Maybe say something like this instead

> (ROSA) makes available an optional logging stack

And also for the other section

> Application logging includes optional log forwarding using the CloudWatch add-on

Maybe use something like

> Application logs may optionally be forwarded using the CloudWatch add-on...

File: modules/rosa-rn-monitoring.adoc [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-monitoring_{context}"]
+= Monitoring
+
+[id="rosa-rn-cluster-metrics_{context}"]
+== Cluster metrics
+
+{product-title} (ROSA) clusters use an integrated Prometheus stack for cluster monitoring, including CPU, memory, and network-based metrics. Access this information through the OpenShift Cluster Manager (OCM) web console. These metrics also allow for horizontal pod autoscaling based on CPU or memory metrics provided by a ROSA customer.
```
Comment (wgordon17) [inline modules/rosa-rn-monitoring.adoc]: > OpenShift Cluster Manager (OCM)

Same comment as above, about including a link to _somewhere_ either _about_ OCM or directly _to_ OCM

File: modules/rosa-rn-monitoring.adoc [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+[id="rosa-rn-cluster-status_{context}"]
+== Cluster status notification
+
+Red Hat communicates the health and status of ROSA clusters through the OCM cluster dashboard and through email notifications sent to the email address of the contact that originally deployed the cluster. Notifications are also sent to the contacts that are added to the Support tab in OCM.
```
Comment (wgordon17) [inline modules/rosa-rn-monitoring.adoc]: > added to the Support tab in OCM

Do we have a document that we can link here about _how_ to do this?

File: modules/rosa-rn-networking.adoc [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-networking_{context}"]
+= Networking
+
+[id="rosa-rn-custom-domains_{context}"]
+== Custom domains for applications
+
+To use a custom hostname for a route, update the DNS provider by creating a canonical name, or `CNAME`, record. The `CNAME` record must map the OpenShift Container Platform canonical router hostname to the custom domain. The canonical router hostname is shown on the _Route Details_ page after a route is created. Alternatively, a wildcard `CNAME` record can be created one time to route all subdomains for a given hostname to the cluster’s router.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc]: I would recommend including an example of what a wildcard CNAME would look like, similarly to https://docs.openshift.com/dedicated/4/getting_started/dedicated-networking.html#dedicated-configuring-your-application-routes_access

File: modules/rosa-rn-networking.adoc [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+[id="rosa-rn-domain-certificates_{context}"]
+== Domain validated certificates
+
+{product-title} (ROSA) includes TLS security certificates that are needed for internal and external routes on the cluster. For more information, see the Service Definition.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc]: > the Service Definition

Include a link?

Comment (wgordon17) [reply modules/rosa-rn-networking.adoc]: > includes TLS security certificates

Probably worth calling out that these are wildcard TLS certificates

File: modules/rosa-rn-networking.adoc [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+
+[id="rosa-rn-custom-certificates_{context}"]
+== Custom certificate authorities for builds
+
+Custom certificate authorities are trusted by builds when images are pulled from an image registry.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc]: Probably worth mentioning that they aren't trusted _automatically_, the CA trust _does_ need to be configured, i.e., https://docs.openshift.com/dedicated/4/builds/setting-up-trusted-ca.html

File: modules/rosa-rn-networking.adoc [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+
+[id="rosa-rn-load-balancers_{context}"]
+== Load Balancers
+
+ROSA supports a number of load balancers that include:
+
+* An internal master load balancer for internal cluster communications traffic
+* An external master load balancing for accessing the APIs. This load balancer can be disabled in the OpenShift Cluster Manager (OCM). When this load balancer is disabled, Red Hat reconfigures the API DNS to point to the internal master load balancer.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc]: > can be disabled in the OpenShift Cluster Manager (OCM)

All of these "can be ... in OCM" can also be done via ROSA CLI

File: modules/rosa-rn-networking.adoc [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+* A default external router/ingress load balancer for application loads, denoted by `apps` in the URL. This load balancer can be configured in OCM to be either publicly or privately accessible. All application routes on the cluster are exposed on this default router load balancer, including cluster services such as the logging UI, metrics API, and registry.
+* An optional, secondary router/ingress load balancer for application loads, denoted by `apps2` in the URL. This load balancer can be configured in OCM to be either publicly accessible or privately accessible. Label matching can be configured so that only application routes matching the label will be exposed on this router load balancer.
+
+[id="rosa-rn-cluster-ingress_{context}"]
+== Cluster ingress
+
+Project administrators can add route annotations for many different purposes, including ingress control through IP whitelisting. Ingress policies can also be changed by using `NetworkPolicy` objects that provide full control over the ingress network policy.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc:43]: Include links to route annotations: https://docs.openshift.com/dedicated/4/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

And NetworkPolicy: https://docs.openshift.com/dedicated/4/networking/network_policy/about-network-policy.html

File: modules/rosa-rn-networking.adoc [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+
+All cluster ingress traffic goes through the defined load balancers. Direct access to all nodes is blocked by cloud configuration.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc]: > All cluster ingress traffic goes through the defined load balancers

This is not 100% accurate, as it's possible to expose a Service via load balancer, which would bypass the defined load balancers and allow for direct TCP access to the backing pods. This is called out in #6 of the Service Definition (https://www.openshift.com/products/dedicated/service-definition#load-balancers)

File: modules/rosa-rn-networking.adoc [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+
+[id="rosa-rn-cluster-egress_{context}"]
+== Cluster egress
+
+Pod egress traffic control through `EgressNetworkPolicy` objects can be used to prevent or limit outbound traffic in ROSA.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc:50]: EgressNetworkPolicy docs: https://docs.openshift.com/dedicated/4/networking/openshift_sdn/configuring-egress-firewall.html

File: modules/rosa-rn-networking.adoc [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+
+OpenShift Container Platform 4 clusters use NAT gateways to present a public, static IP for any public outbound traffic leaving the cluster. Each availability zone a cluster is deployed into receives a distinct NAT gateway. Any traffic that remains inside the cluster, or does not go out to the public Internet, does not pass through the NAT gateway and, instead, has a source IP address belonging to the node that the traffic originated from. Node IP addresses are dynamic.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc:52]: Since this is ROSA, I would recommend calling out that the Elastic IPs (EIP) can be looked up directly in the AWS console.

File: modules/rosa-rn-networking.adoc [hunk 19]
```diff
@@ -0,0 +1,63 @@
...
+* AWS VPC peering
+* AWS Transit Gateway
+* AWS Direct Connect
+
+The Red Hat site reliability engineers (SREs) do not monitor private network connections.
+
+[id="rosa-rn-dns-forwarding_{context}"]
+== DNS forwarding
+
+For ROSA clusters that have a private cloud network configured, customers can specify internal DNS servers available on that private connection that should be queried for explicitly provided domains.
```
Comment (wgordon17) [inline modules/rosa-rn-networking.adoc:71]: Probably worth including a link about DNS forwarding: https://docs.openshift.com/dedicated/4/networking/dns-operator.html#nw-dns-forward_dns-operator

And also calling out that customers can set up DNS forwarding via Route53 rules as well

File: modules/rosa-rn-platform.adoc [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-platform_{context}"]
+= Platform
+
+[id="rosa-rn-backups_{context}"]
+== Cluster backups
+
+Application and application data backups are not a part of the {product-title} (ROSA) service. Customers must have their own backup plan for their applications and application data.
```
Comment (wgordon17) [inline modules/rosa-rn-platform.adoc]: > backup plan

I would recommend saying

> strategy

As "plan", to me, implies that there's some kind of backup service that RH/AWS offers that I can/should purchase.

File: modules/rosa-rn-platform.adoc [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+
+All Kubernetes objects and persistent volumes (PVs) in each ROSA cluster are backed up to facilitate a prompt recovery in the unlikely event that a cluster becomes irreparably inoperable. These backups are stored in a secure object storage (multiple availability zone) bucket in the same account as the cluster.
+
+Node root volumes are not backed up because Red Hat Enterprise Linux CoreOS is fully managed by the OpenShift Container Platform cluster and no stateful data should be stored on a node’s root volume.
+
+[id="rosa-rn-autoscaling_{context}"]
+== Autoscaling
+
+Autoscaling is available through the `rosa` command-line. ROSA supports autoscaling on nodes. Autoscaling can be configured during cluster creation or on machine pools after a cluster is created.
```
Comment (wgordon17) [inline modules/rosa-rn-platform.adoc:21]: I _believe_ that autoscaling is also available in OCM

File: modules/rosa-rn-platform.adoc [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+
+[id="rosa-rn-multi-az_{context}"]
+== Multiple availability zone
+
+In a multiple availability zone cluster, master nodes are distributed across availability zones and one worker node is required in each availability zone.
+
+[id="rosa-rn-node-labels_{context}"]
+== Node labels
+
+Custom node labels are created by Red Hat during node creation and cannot be changed on ROSA clusters at this time.
```
Comment (wgordon17) [inline modules/rosa-rn-platform.adoc]: Node labels _can be_ defined when creating a MachinePool, but they cannot be edited once created

File: modules/rosa-rn-platform.adoc [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+
+[id="rosa-rn-versions_{context}"]
+== OpenShift Container Platform Versions
+
+ROSA runs as a service on Amazon Web Services (AWS) and is kept up-to-date with the latest OpenShift Container Platform version.
+
+[id="rosa-rn-upgrades_{context}"]
+== Upgrades
```
Comment (wgordon17) [inline modules/rosa-rn-platform.adoc:44]: Please direct to the Life Cycle page: https://access.redhat.com/support/policy/updates/openshift/dedicated

File: modules/rosa-rn-platform.adoc [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+
+[id="rosa-rn-operating-system_{context}"]
+== Operating system
+
+ROSA runs on OpenShift Container Platform 4 and uses Red Hat Enterprise Linux CoreOS as the operating system for all master and worker nodes.
+
+[id="rosa-rn-kubernetes-os_{context}"]
+== Kubernetes operator support
+
+ROSA supports privileged Operators created by Red Hat and Certified ISVs.
```
Comment (wgordon17) [inline modules/rosa-rn-platform.adoc]: This is no longer true, and has been updated on https://www.openshift.com/products/dedicated/service-definition#operator-support

File: modules/rosa-rn-security.adoc [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+* GitHub or GitHub Enterprise
+* GitLab
+* Google
+* LDAP
+* OpenID connect
+
+[id="rosa-rn-priv-containers_{context}"]
+== Privileged containers
+
+Privileged containers are supported on ROSA with the `cluster-admin` role. To enable Red Hat to operate ROSA as a managed service with an service level agreement (SLA), some restrictions are enforced to limit the ability of rogue or accidental changes that could impact the service.
```
Comment (wgordon17) [inline modules/rosa-rn-security.adoc]: This is _also_ no longer true, and has been updated in https://www.openshift.com/products/dedicated/service-definition#privileged-containers

**Review by wgordon17 — CHANGES_REQUESTED** [review]
All of this seems _very much_ more appropriate in the Service Definition, I'm not sure they count as "release features". For the most part, it seems we're just parroting what should already be in the Service Definition, unless that's the point? ¯\_(ツ)_/¯

File: modules/rosa-rn-platform.adoc (hunk 21 above)
Comment (jeremyeder) [reply modules/rosa-rn-platform.adoc:21]: @vkareh autoscaler pushed to prod OCM UI?

Comment (vkareh) [reply modules/rosa-rn-platform.adoc:21]: nope, in the OCM web UI this is still WIP

File: assemblies/rosa-rn-about.adoc (hunk 1 above)
Comment (0kashi) [reply assemblies/rosa-rn-about.adoc]: It is RHCOS AFAIK

File: modules/rosa-rn-acct-management.adoc (hunk 4 above)
Comment (0kashi) [reply modules/rosa-rn-acct-management.adoc]: I also think the wording here is a bit off.  "Usage" and "used" are about the same topic.  Maybe rephrase to "...base on the usage of AWS components by the service...."?

File: modules/rosa-rn-acct-management.adoc (hunk 6 above)
Comment (0kashi) [reply modules/rosa-rn-acct-management.adoc]: I think you can mix.  In one machine pool they have to be the same, but if you create two pools with different instance types, that works just fine....at least as far as I tested...

File: modules/rosa-rn-acct-management.adoc (hunk 7 above)
Comment (0kashi) [reply modules/rosa-rn-acct-management.adoc]: I think as part of the process they need to create/have/identify their Red Hat account otherwise access won't be able to be granted within OCM.  I am not sure what the workflow is though if someone tries to create a ROSA cluster and does not have a red hat account. @acathrow should know.

File: modules/rosa-rn-security.adoc [hunk 26]
```diff
@@ -0,0 +1,55 @@
...
+== Authentication provider
+
+Authentication for the cluster can be configured using either the link:https://cloud.redhat.com/openshift[OpenShift Cluster Manager (OCM)] cluster creation process or using the `rosa` CLI.
+
+OpenShift Container Platform is not an identity provider, and all access to the cluster must be managed by customers as part of their integrated solution. {product-title} (ROSA) supports the use of multiple identity providers that are simultaneously provisioned.
+
+The following identity providers are supported:
+
+* GitHub or GitHub Enterprise
+* GitLab
```
Comment (0kashi) [inline modules/rosa-rn-security.adoc:19]: @wgordon17 - Is GitLab supported on ROSA?  I don't think it is on OSD...

File: modules/rosa-rn-storage.adoc [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+
+[id="rosa-rn-encrypted-pv_{context}"]
+== Encrypted-at-rest PV
+
+By default, EBS volumes used for persistent volumes (PVs) are encrypted-at-rest.
+
+[id="rosa-rn-block-storage_{context}"]
+== Block storage (RWO)
+
+PVs are backed by Amazon Web Services (AWS) block storage, known as AWS EBS, that is Read-Write-Once.
```
Comment (0kashi) [inline modules/rosa-rn-storage.adoc:22]: @wgordon17 - Although not fully supported do we want to include here the ability to use EFS and how one would do so?   Someone looking for storage docs might want to do RWX as well...

File: assemblies/rosa-rn-new-features.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply assemblies/rosa-rn-new-features.adoc:13]: Linking may work differently in PV2. That's something I need to research.

File: modules/rosa-rn-security.adoc (hunk 26 above)
Comment (wgordon17) [reply modules/rosa-rn-security.adoc:19]: Yup, I see it in the ROSA CLI

```
Type of identity provider. Options are [github gitlab google ldap openid]
```

File: modules/rosa-rn-storage.adoc (hunk 27 above)
Comment (wgordon17) [reply modules/rosa-rn-storage.adoc:22]: If we _were_ to mention it, we'd probably want to update the KCS article to make it more generalized toward both OSD _and_ ROSA https://access.redhat.com/articles/5025181

File: assemblies/rosa-rn-about.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply assemblies/rosa-rn-about.adoc]: @wgordon17 Yes, the Service Definition is what was suggested to me to use as a basis for the RN, because it. Normally for Release Notes we'd just document the new features. But this is a first release and it seems appropriate to point out a little bit more about ROSA. Feel free to point out anything that you think should be removed though. I suspect this content might be too much info.

File: assemblies/rosa-rn-new-features.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply assemblies/rosa-rn-new-features.adoc:13]: I can add a hard coded link after everything is published.

File: modules/rosa-rn-about.adoc (hunk 3 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-about.adoc]: I reworded it.

Comment (jonquilwilliams) [reply modules/rosa-rn-about.adoc]: I was told but PM that someone (PM?) is working on an OCP/ROSA comparison matrix. I haven't heard any updates yet. I think the Service Definition might be their best resource for what is managed by RH & AWS. That document is referenced earlier.

File: modules/rosa-rn-acct-management.adoc (hunk 4 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-acct-management.adoc]: Wording tweaked, thanks. 

@wgordon17 I've added wording about the ROSA CCS document, but can't add a hard coded link until after it's published.

File: modules/rosa-rn-acct-management.adoc (hunk 5 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-acct-management.adoc]: Added the link for OCM.

File: modules/rosa-rn-acct-management.adoc (hunk 6 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-acct-management.adoc]: Fixed.

File: modules/rosa-rn-acct-management.adoc (hunk 7 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-acct-management.adoc]: Since this is the Release Notes, I don't think we need to get into that level of granularity. Other documents provide the Support details, like the Service Definition, and Andy said not to tie the Support info to the account info. To use ROSA, they have to create an RH account to log in and use the service anyway, which is detailed in the user docs.

I'm leaving this as-is.

File: modules/rosa-rn-logging.adoc (hunk 8 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-logging.adoc]: Done.

File: modules/rosa-rn-networking.adoc (hunk 11 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc]: Done.

File: modules/rosa-rn-monitoring.adoc (hunk 10 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-monitoring.adoc]: Not yet. We have another writer outside of our team who is actively working on generic OCM documentation that we will take over and bring into managed services. This will not be in time for GA. Not sure of the ETA for it.

File: modules/rosa-rn-monitoring.adoc (hunk 9 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-monitoring.adoc]: Link added to OCM.

File: modules/rosa-rn-networking.adoc (hunk 12 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc]: Added "wildcard" to TLS certificates. Have to wait until publishing to get a link for the Service Definition.

File: modules/rosa-rn-networking.adoc (hunk 13 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc]: Done.

File: modules/rosa-rn-networking.adoc (hunk 14 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc]: Done.

File: modules/rosa-rn-networking.adoc (hunk 15 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc:43]: The ROSA doc set will include its own copy of these topics. Link will have to be added after publishing because we don't have ROSA URLs yet.

File: modules/rosa-rn-networking.adoc (hunk 16 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc]: I think that by "defined load balancers", the Service Definition, from which this was taken, is including the optional load balancer for services. But I added a clarifier here.

File: modules/rosa-rn-networking.adoc (hunk 17 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc:50]: ROSA will have its own version of network policy docs, and I'll have to add a hard coded link after it's published.

File: modules/rosa-rn-networking.adoc (hunk 18 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc:52]: Done.

File: modules/rosa-rn-networking.adoc (hunk 19 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-networking.adoc:71]: Added the Route 53 option.

ROSA will have its own version of network policy docs, and I'll have to add a hard coded link after it's published.

File: modules/rosa-rn-platform.adoc (hunk 20 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-platform.adoc]: Ah, good point. Done.

File: modules/rosa-rn-platform.adoc (hunk 21 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-platform.adoc:21]: We're not pushing customers to OCM in this release. The focus is on documenting the CLI.

File: modules/rosa-rn-platform.adoc (hunk 22 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-platform.adoc]: OK, this has been changed. I rechecked the `rosa create cluster` command in Github. Thanks.

File: modules/rosa-rn-platform.adoc (hunk 23 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-platform.adoc:44]: Done.

File: modules/rosa-rn-platform.adoc (hunk 24 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-platform.adoc]: Fixed.

File: modules/rosa-rn-security.adoc (hunk 25 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-security.adoc]: Fixed. Need to add a link to the Service Definition when it is published.

File: modules/rosa-rn-security.adoc (hunk 26 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-security.adoc:19]: Thanks for checking. I'll leave it as is then.

File: modules/rosa-rn-storage.adoc (hunk 27 above)
Comment (jonquilwilliams) [reply modules/rosa-rn-storage.adoc:22]: @wgordon17  @okashi18  Please let me know if you're planning to update that PCS article and whether you want to include a general comment about the ability to use EFS. Thanks.

File: modules/rosa-rn-logging.adoc [hunk 28]
```diff
@@ -0,0 +1,11 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-logging_{context}"]
+= Logging
+
+{product-title} (ROSA) offers an optional logging stack based on Elasticsearch, Fluentd, and Kibana (EFK). The logging stack in OpenShift Container Platform is designed for short-term retention to aid application troubleshooting, not for long-term log archiving.
```
Comment (jeremyeder) [inline modules/rosa-rn-logging.adoc:9]: I think we should drop this sentence on EFK in order to focus on CloudWatch.

File: modules/rosa-rn-logging.adoc [hunk 29]
```diff
@@ -0,0 +1,11 @@
...
+
+Application logging are supported. Application logs can be forwarded using the CloudWatch add-on service. Add-on services are configured using the `rosa` command-line.
```
Comment (jeremyeder) [inline modules/rosa-rn-logging.adoc:11]: I think  you should drop the first bit here.  So just ...  Logs can optionally be forwarded to CloudWatch.  This can be configured using the `rosa` command-line.
